cluster-autoscaler/Makefile

```diff
@@ -20,6 +21,9 @@ ifdef LDFLAGS
 else
   LDFLAGS_FLAG=
 endif
+ifdef DOCKER_CLEANUP
+  RM_FLAG=--rm
```

Comment:
To be consistent with other flags, can you set `RM_FLAG` to empty in `else`? Moreover, can you rename it to `DOCKER_CLENUP_FLAG`. Or rename  `DOCKER_CLENUP` to `DOCKER_RM` so that we don't have both `CLEANUP`and `RM`.

Reply:
I went with `DOCKER_RM` to be consistent with `DOCKER_NETWORK`